### DISCUSSION TITLE
Docker安装 需要进入debug模式 结果报错

### DISCUSSION THREAD

**Discussion by gzxiexl** [AlistGo/alist#5210] [General]
FATA[2023-09-13 17:11:45]/app/cmd/server.go:55 github.com/alist-org/alist/v3/cmd.glob..func8.1() failed to start http: listen tcp 0.0.0.0:5244: bind: address already in use

**Comment by anwen-anyi** [comment]
AList已经启动了你想办法停止了，在启动。 试试下面的可以吗？我不确定

```log
docker exec -it alist ./alist stop
```

```log
docker exec -it alist ./alist server --debug
```
